ql/src/test/org/apache/hadoop/hive/ql/exec/tez/TestTezOutputCommitter.java

```diff
@@ -122,6 +122,7 @@ private IDriver getDriverWithCommitter(String committerClass) {
     conf.setVar(HiveConf.ConfVars.HIVE_AUTHORIZATION_MANAGER,
         "org.apache.hadoop.hive.ql.security.authorization.plugin.sqlstd.SQLStdHiveAuthorizerFactory");
     conf.setBoolVar(HiveConf.ConfVars.HIVE_SUPPORT_CONCURRENCY, false);
+    conf.setBoolVar(HiveConf.ConfVars.HIVESTATSCOLAUTOGATHER, false);
```

Comment:
Why is this required here?

Reply:
Otherwise the tests are failing, because with stats turned on we generate 2 tasks instead of 1 (change of the execution plans which contain a stage)